tests/conftest.py

```diff
@@ -160,3 +161,12 @@ def metek_raw_gz_file(tmp_path_factory):
         with open(fnameo, "wb") as fout:
             shutil.copyfileobj(fin, fout)
     return fnameo
+
+
+@pytest.fixture
```

Comment:
Nice @egouden, would it make sense to create a second fixture which directly creates the name (for single file use case)?

```python
@pytest.fixture
def temp_file(tmp_path):
    rnd = uuid.uuid4().hex[:8]
    return (tmp_path / f"testfile_{rnd}").name
```

Reply:
done